upgrade-notes/backend_20240717_111911.md

```diff
@@ -0,0 +1,3 @@
+#### limited item counts in sliders ([#3244](https://github.com/shopsys/shopsys/pull/3244))
+
+-   see #project-base-diff to update your project
```

Comment:
can you please mention changes in the classes? renamed methods, new arguments, etc.